dashboard/app/controllers/api/v1/amazon_future_engineer_controller.rb

```diff
@@ -29,6 +30,20 @@ def submit
       new_code_account: current_user.created_at > 5.minutes.ago
     )
 
+    FirehoseClient.instance.put_record(
+      {
+        study: 'amazon-future-engineer-eligibility',
+        event: 'submit_to_afe',
+        data_json: {
+          accountEmail: current_user.email,
+          accountSchoolId: current_user&.school_info&.school&.id,
+          formEmail: afe_params['email'],
+          formSchoolId: afe_params['schoolId'],
+          formData: submission_body
+        }.to_json
+      }
+    )
```

Comment:
Right now we're tracking this event if we successfully submit to AFE _or_ if submitting to AFE is disabled for that environment. (But not if we skip submission or submission fails for some other reason.)  That seems okay - is it the desired behavior?

Reply:
Yeah it was sort of for testing purposes, so I could see what would be sent to Firehose in development (since it sends Firehose requests to your logfile). Seems maybe useful in the future as well, to have the entire payload returned to the controller in development (instead of just immediately returning)?